src/app/beer_garden/router.py

```diff
@@ -574,22 +576,18 @@ def _determine_target_garden(operation: Operation) -> str:
 def _forward_stomp(operation: Operation, target_garden: Garden):
     try:
         if not stomp_garden_connections[target_garden.name].is_connected():
-            stomp_garden_connections[target_garden.name].connect(
-                username=target_garden.connection_params["stomp_username"],
-                passcode=target_garden.connection_params["stomp_password"],
-                wait=True,
-                headers={
-                    "client-id": target_garden.connection_params["stomp_username"]
-                },
-            )
+            if not stomp_garden_connections[target_garden.name].connect():
```

Comment:
Assuming that this returns back a True/False, this looks good. If it returns anything else then we should look more into it.

Reply:
Yes, part of this PR is that the router (specifically `stomp_garden_connections`) uses our Connection class instead of the stomp.py one. That method returns a boolean.

Reply:
The UI stuff is not purely formatting. The `ssl` section has been changed from:

* use_ssl
* cert_file
* private_key
* verify_host
* verify_hostname

To:

* use_ssl
* ca_cert
* verify_host
* verify_hostname
* client_cert
* client_key